operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -573,6 +573,7 @@ func TestConfigSelectors(t *testing.T) {
 		// Next we fetch all the objects for a revision install
 		nameRev := "istiod-canary"
 		deploymentRev := mustFindObject(t, objsRev, nameRev, name.DeploymentStr)
+		hpaRev := mustFindObject(t, objs, nameRev, name.HPAStr)
```

Comment:
@totallyGreg  After spending an hour trying to debug a test, found out that you were looking into the wrong object for the Rev label. I ran it locally and now  `make gen` and `make refresh-goldens` are passing.
```suggestion
		hpaRev := mustFindObject(t, objsRev, nameRev, name.HPAStr)
```